ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh

```diff
@@ -61,7 +61,7 @@ v414=" ${v413} MachineAPI Build DeploymentConfig ImageRegistry"
 v415=" ${v414} OperatorLifecycleManager CloudCredential"
 v416=" ${v415} CloudControllerManager Ingress"
 v417=" ${v416}"
-v418=" ${v417}"
+v418=" ${v417} OperatorLifecycleManagerV1"
```

Comment:
Beside these two step need to be updated, the following step also need to be updated:
```
ipi-conf-capability-additionalcaps
ipi-conf-capability-baselinecaps-vset
cucushift-upgrade-check-capability
```

Reply:
> ipi-conf-capability-additionalcaps
> ipi-conf-capability-baselinecaps-vset

Added them.
> cucushift-upgrade-check-capability

I didn't find the related config for 4.17 and 4.18 there, not sure if any specific configs, ignored them in this PR, left it to the step owner.